src/utils.ts

```diff
@@ -558,6 +558,20 @@ export function thirtyDaysFromNow(): Date {
   return new Date(Date.now() + THIRTY_DAYS_IN_MILLISECONDS);
 }
 
+/**
+ * Get the current version of Node.js engine from `process.version`.
+ * @return Node.js major release version
+ */
+export function getNodeVersionString(): string {
+  const nodeVersion: RegExpExecArray | null = /\d+/.exec(process.version);
+
+  if (nodeVersion === null) {
+    throw new Error("Node version is not found");
```

Comment:
Instead of throwing an error, let's return some sane default value like `14`.

Reply:
I can add a constant for the default value, i.e. `const DEFAULT_NODE_VERSION = 14`, and return it if it seems good enough for you.